operator/src/main/java/org/keycloak/operator/crds/v2alpha1/realmimport/PlaceholderSecret.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.operator.crds.v2alpha1.realmimport;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.fabric8.kubernetes.api.model.SecretKeySelector;
+import io.sundr.builder.annotations.Buildable;
+
+import java.util.Objects;
+
+/**
+ * @author Scott Tustison
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Buildable(editableEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder")
+public class PlaceholderSecret {
```

Comment:
I can imagine a few different situations here.

1. Support **only secrets** for placeholders - If we want to support only secrets, we might not need the new object `PlaceholderSecret`, but only the `SecretKeySelector`, right? 
2. Support **secrets** and **config maps** for placeholders - If we want to support both of them and have them both in the same object, I'd choose a different name for the class. 

I think the 2nd option is more accurate, so I'd propose only this nitpick related to changing the class name for something like this:
```suggestion
public class Placeholder {
```

WDYT?

Reply:
@mabartos seems fine to change the class name.